opensrp-fp/src/main/java/org/smartregister/fp/common/util/Utils.java

```diff
@@ -1120,4 +1118,61 @@ public static void openMECWheelApp(Context context) {
         }
         context.startActivity(intent);
     }
+
+    public static LocalDate getNextContactVisitDate(JSONObject jsonObject) {
+        String methodExit = getJsonFieldValue(jsonObject, "step7", "method_exit");
+
+        DateTimeFormatter pattern = DateTimeFormat.forPattern(("dd-MM-yyyy"));
+        LocalDate nextContactDate = null;
+        switch (methodExit) {
+            case "cu_iud":
+            case "lng_iud":
+                nextContactDate = LocalDate.parse(getJsonFieldValue(jsonObject,"step7", "iud_insertion_date"), pattern);
```

Comment:
Please move the strings to a constants file

Reply:
Done